frontend/src/tests/components/common/RenderTime.spec.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { shallow } from "enzyme";
+import moment from "moment";
+import RenderTime from "@/components/common/RenderTime";
+
+let props = {};
+
+const setupProps = () => {
```

Comment:
Is it possible to get a unit test on the new functions like parseFormDataIntoPayload?  They seem like a worthwhile place to add some coverage